hack/local-up-karmada.sh

```diff
@@ -29,12 +29,18 @@ KIND_LOG_FILE=${KIND_LOG_FILE:-"/tmp/karmada"}
 #step0: prepare
 # proxy setting in China mainland
 
-if [[ -n ${CHINA_MAINLAND:-} ]]; then
-export GOPROXY=https://goproxy.cn # set go proxy
+if [[ ! -n ${CHINA_MAINLAND:-} ]]; then
```

Comment:
Is it a bug? @lfbear

Reply:
I think we should not use China-mainland proxy mode default, I still suggest using it explicitly by `export CHINA_MAINLAND=true` @Phil-sun how about it

Reply:
l'm so sorry, l misunderstood this code.  Using China-mainland proxy mode by ```export CHINA_MAINLAND=true``` is a good way. Maybe we can add this into quick start for China mainland users. @lfbear

Reply:
I don't think so. We shouldn't highlight `China-mainland` heavily, this info isn't necessary for other users.